### DISCUSSION TITLE
Which race track has the nickname "Brickyard"? 🏎️🏁

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1278] [Galaxy Trivia] [answered]
**Bonus:** Which company made DeSoto motor vehicles? 🚗

**Comment by tomerf** [comment]
The Indianapolis Motor Speedway is nicknamed "The Brickyard".

Bonus: DeSoto was an American automobile made by Chrysler Corporation.

**Comment by peterbaccaro** [comment, marked as answer]
The racetrack named "The Brickyard" is the **Indianapolis Motor Speedway**, known for its original brick surface.

DeSoto was a brand of motor vehicles and a division of the **Chrysler Corporation**.  The DeSoto brand was produced between 1928 and 1961.
